### DISCUSSION TITLE
The combination Left Shift+Left Alt+Right Arrow does not work after remaping the keys on new laptop Lenovo E15 gen 4 Win 10

### DISCUSSION THREAD

**Discussion by locust2014** [houmain/keymapper#182] [Q&A] [answered]
The combination Left Shift+Left Alt+Right Arrow does not work after remaping the keys on new laptop Lenovo E15 G4 (Windows 10).

List of keys remapped:

MetaLeft >> AltLeft
AltLeft >> MetaLeft
MetaRight >> AltRight
AltRight >> MetaRight

This remapping is necessary for the Mac keyboard.

Reinstalling the Windows operating system did not solve the problem.

Reinstalling the drivers and installing the latest drivers did not solve the problem.

The problem does not appear on my old Lenovo ThinkPad T590 laptop.

Could I work around this problem with some additional remaps that I do not know?

Please advise a workaround.

**Comment by ristomatti** [comment, marked as answer]
Have you checked if the key combination works w/o Keymapper running e.g. here https://keyevents.netlify.app/? If the same config worked with your previous keyboard but doesn't on the new one, it might be an unfortunate hardware limitation, see discussion at #171.
